tests/function/test_service.py

```diff
@@ -134,9 +134,7 @@ def test_virtwho_service_control_by_ssh_connect(self, virtwho, ssh_guest, ssh_ho
         assert status == "dead"
 
         # start
-        ssh_guest.runcmd(
-            f"ssh {server} -p {port} " f'"systemctl restart virt-who"'
-        )
+        ssh_guest.runcmd(f"ssh {server} -p {port} " f'"systemctl restart virt-who"')
```

Comment:
```suggestion
        ssh_guest.runcmd(f"ssh {server} -p {port} systemctl restart virt-who")
```

Reply:
Seems this would be better. What do you think of this?

Reply:
updated to `ssh_guest.runcmd(f"ssh {server} -p {port} 'systemctl restart virt-who'")`
but after testing the `ssh_guest.runcmd(f"ssh {server} -p {port} systemctl restart virt-who")` also works.
thanks.

Reply:
thanks for the quick reply